src/serialize/window.js

```diff
@@ -45,6 +45,10 @@ function getSerializedWindow(winPromise : ZalgoPromise<CrossDomainWindowType>, {
     return {
         id,
         getType: () => winPromise.then(win => {
+            if (isWindowClosed(win)) {
```

Comment:
Can we still try to determine the type when serialize is first called? That at least means getType() will continue to work so long as the window was at least open when it was first serialized.